src/test/TestCommon.h

```diff
@@ -0,0 +1,41 @@
+/*
+* Copyright (c) 2013-2023 Rony Shapiro <ronys@pwsafe.org>.
+* All rights reserved. Use of the code is allowed under the
+* Artistic License 2.0 terms, as specified in the LICENSE file
+* distributed with this code, or available from
+* http://www.opensource.org/licenses/artistic-license-2.0.php
+* Contributed by Ashley R. Thomas, 05-Oct-2023
+*/
+// TestCommon.h
+//-----------------------------------------------------------------------------
+
+#pragma once
```

Comment:
This isn't portable. Please wrap the contents in the standard
#ifndef _FILENAME_H
#define _FILENAME_H
...
#endif
instead